gradle.properties

```diff
@@ -28,22 +28,12 @@
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 
 org.gradle.daemon=true
+org.gradle.configureondemand=true
 org.gradle.parallel=true
 org.gradle.caching=true
 
 org.gradle.jvmargs=-Xmx1g
-
 org.gradle.workers.max=8
 
 android.useAndroidX=true
 android.enableJetifier=true
-
-android.enableDexingArtifactTransform.desugaring=true
```

Comment:
Are these things not needed anymore?

Reply:
This flag is on by default, so it can be removed.